### ISSUE TITLE
14.3.2 (Flow 2.3.2) broke live reload

### ISSUE THREAD

**Issue by jojule** [vaadin/flow#8814]
Vaadin 14.3.2 broke live reload altogether. When reloading, the it it produces:
<img width="755" alt="Screen Shot 2020-08-04 at 8 02 33 PM" src="https://user-images.githubusercontent.com/644969/89367275-7337f000-d68d-11ea-9671-be7e8e3e7225.png">
(Both on Chrome and Safari)

Here is how to reproduce: 
https://drive.google.com/file/d/1WvlEurO9n91XFmaKWecyhmanG4237N4p/view

**Comment by jojule** [comment]
Tested that exact same scenario worked on 14.3.1 on same setup

**Comment by denis-anisimov** [comment]
The error is absolutely cryptic for me and I have no any idea why it happens.
This is black JS magic which I don't understand in absolutely trivial code which should just work.
It's related somehow to "temporal dead zone" but I don't understand how.

So I've just replaced `let`  with `var` and hope that it works as expected.

I would like avoid releasing this until it's tested. So we will make a SNAPSHOT today. 
Could you please test it with the SNAPSHOT ?

**Comment by Artur-** [comment]
The JS code that is executed in 14.3.2 looks like this
```
      const load = function() {
          if (iframe.contentWindow.Vaadin) {
              const contentIsPopulated = contentIsPopulated = iframe.contentDocument.body.children.length >= document.body.children.length;
              if (contentIsPopulated && !iframe.contentWindow.expired) {
                  setTimeout(transferIframe, 100);
              } else {
                  setTimeout(load, 100);
              }
```

**Comment by Artur-** [comment]
Using `2.3-SNAPSHOT` does not change anything. Live reload is always broken

**Comment by denis-anisimov** [comment]
Shit, 
I'm extremely tired of 1000 branches and versions.

The fix is done in the master

**Comment by denis-anisimov** [comment]
Fix for #8729 will be reverted in `2.3` at least.
That will fix this ticket but return back #8729.

**Comment by pleku** [comment]
14.3.3 should have this one fixed now.
